src/NumericalAlgorithms/Spectral/SwshCoefficients.hpp

```diff
@@ -8,9 +8,88 @@
 #include <sharp_cxx.h>
 
 #include "NumericalAlgorithms/Spectral/SwshCollocation.hpp"
+#include "Utilities/ForceInline.hpp"
 
 namespace Spectral {
 namespace Swsh {
+
+/// \ingroup SwshGroup
+/// \brief Convenience function for determining the number of spin-weighted
+/// spherical harmonics coefficients that are stored for a given `l_max`
+constexpr SPECTRE_ALWAYS_INLINE size_t
+number_of_swsh_coefficients(const size_t l_max) noexcept {
+  return (l_max + 1) * (l_max + 2) / 2;  // "triangular" representation
+}
+
+/*!
+ * \ingroup SwshGroup
+ * \brief Compute the relative sign change necessary to convert between the
+ * libsharp basis for spin weight `from_spin_weight` to the basis for spin
+ * weight `to_spin_weight`, for the real component coefficients if `real` is
+ * true, otherwise for the imaginary component coefficients.
+ *
+ * \details The sign change is obtained by the
+ * difference between the libsharp convention and the convention which uses:
+ * \f[
```

Comment:
Possible ping for spacing in this file (2x) if you care. Feel free to click resolve on this comment if you don't :)